integrationTests/chainSimulator/common.go

```diff
@@ -23,6 +23,8 @@ const (
 	minGasPrice                             = 1000000000
 	txVersion                               = 1
 	mockTxSignature                         = "sig"
+	deposit                                 = "deposit"
+	multiEsdtTransfer                       = "MultiESDTNFTTransfer"
```

Comment:
We already have this const for this:
`core.BuiltInFunctionMultiESDTNFTTransfer`